nautobot/core/views/routers.py

```diff
@@ -79,4 +79,22 @@ class NautobotUIViewSetRouter(SimpleRouter):
             detail=True,
             initkwargs={"suffix": "Edit"},
         ),
+        Route(
+            url=r"^{prefix}/{lookup}/changelog/$",
+            mapping={
+                "get": "changelog",
+            },
+            name="{basename}_changelog",
+            detail=True,
+            initkwargs={"suffix": "Changelog"},
+        ),
+        Route(
+            url=r"^{prefix}/{lookup}/notes/$",
+            mapping={
+                "get": "notes",
+            },
+            name="{basename}_notes",
+            detail=True,
+            initkwargs={"suffix": "Notes"},
+        ),
```

Comment:
I'm curious now how `DeviceRedundancyGroup` will be affected by this. Tests didn't fail but these routes appear they might be registered twice now for that model. @smk4664 cleaned them up for Circuits.

Reply:
Seems like we should fix this now before merging.

Reply:
Ya?

```diff
diff --git a/nautobot/dcim/urls.py b/nautobot/dcim/urls.py
index 8d70e92fb..f33cf3a7f 100644
--- a/nautobot/dcim/urls.py
+++ b/nautobot/dcim/urls.py
@@ -1635,17 +1635,5 @@ urlpatterns = [
         name="powerfeed_connect",
         kwargs={"termination_a_type": PowerFeed},
     ),
-    path(
-        "device-redundancy-groups/<slug:slug>/changelog/",
-        ObjectChangeLogView.as_view(),
-        name="deviceredundancygroup_changelog",
-        kwargs={"model": DeviceRedundancyGroup},
-    ),
-    path(
-        "device-redundancy-groups/<slug:slug>/notes/",
-        ObjectNotesView.as_view(),
-        name="deviceredundancygroup_notes",
-        kwargs={"model": DeviceRedundancyGroup},
-    ),
 ]
 urlpatterns += router.urls
(END)
```